sdks/java/extensions/euphoria/euphoria-beam/build.gradle

```diff
@@ -35,6 +35,8 @@ dependencies {
     testCompile project(':beam-runners-direct-java')
     testCompile library.java.slf4j_api
     testCompile library.java.hamcrest_core
+    testCompile 'org.slf4j:slf4j-simple:1.7.+'
```

Comment:
we should get rid of all hardcoded dependencies. What are other beam modules using for logging in tests?